target_postgres/postgres.py

```diff
@@ -9,6 +9,8 @@
 import arrow
 from psycopg2 import sql
 
+import target_postgres.json_schema as JSONSchema
```

Comment:
I actually think keeping it as `json_schema` would be more pythonic. Capitalized words are generally used for classes, which below made me thing you had created a JSONSchema class.

Reply:
Yeah, the reason I _didn't_ do that here was that the variable `json_schema` is...everywhere...in this file. Due to this, I figured it'd be far simpler to have a separate PR which only makes the correction to the names as a pure refactor, thus keeping the meat of this PR tiny and focused. I'm happy to make that change here as well in a single comment!

Reply:
Hmm true. I also use the variable `json_schema` a lot in taps. We could rename the module to something like `json_schema_utils` or import the functions directly.